exporters/opensearch-exporter/pom.xml

```diff
@@ -189,6 +189,7 @@
         <configuration>
           <ignoredNonTestScopedDependencies>
             <ignoredNonTestScopedDependency>com.fasterxml.jackson.core:jackson-core</ignoredNonTestScopedDependency>
+            <ignoredNonTestScopedDependency>software.amazon.awssdk:regions</ignoredNonTestScopedDependency>
```

Comment:
I don't know why but yesterday I was struggling with this library, now everything works as expected. Thanks for the comment 😄

Reply:
I'm reopening this conversation because I can't change the `scope` of this library to `test` as it is used by the new library that I add.

But if I leave it without scope (compile by default) I got this error:
`
[ERROR] Non-test scoped test only dependencies found:
[ERROR]    software.amazon.awssdk:regions:jar:2.20.104:compile
`
This is a strange behaviour from maven, the only solution that I found is to add it within the ignored non-test dependencies

Reply:
I can see 2 other solutions. Either we change the `RestClientFactory` to create the `AwsRequestSigningApacheInterceptor` with a `Region` instead of a `String`. Then we are using it in our non-test code.
Alternatively we could a `String` instead of a `Region` in the `AwsRequestSigningApacheInterceptorTest`. Then we are no longer using the dependency at all and we can get rid of it.

The second one would be my preference.